File: intetest/0r_record.c
#include<stdio.h>
#include<stdbool.h>
struct books{
int title[2];
};
struct books b;
int test(struct books a)
{
int _test_;
_test_ = a.title;
return _test_;
}
int main() {
b.title = 1;
printf("%d",b.title);
printf("%d",test(b));
return 0;
}

File: intetest/0r_record.pas
program main;
type
    books = record
    title:array [0..1] of integer;
end;
type
    stu=record
    bo:books;
end;
var b:books;c:stu;d:integer;
function test(a:books):integer;
begin
    test:=a.title[0];
end;
begin
    b.title[0]:=d;
    d:=c.bo.title;
    {write(b.title[0]);
    write(test(b));}
end.
